docs/book/user-guide/advanced-guide/data-management/model-management.md

```diff
@@ -323,5 +323,71 @@ artifact, and you specify whether it's for a model or deployment by using the
 `is_model_artifact` and `is_deployment_artifact` flags (as shown above) else it
 will be assumed to be a data artifact.
 
+### Produce intermediate artifacts
+
+It often comes handy to save some of your work half-way: steps like epoch-based training can be running slow and you don't want to loose any checkpoints along the way, if error occurs.
+You can use `save_artifact` utility function to save your data assets as ZenML artifact. Moreover, if your step has Model context configured in `@pipeline` or `@step` decorator it will be automatically linked to it, so you can get easy access to it using Model Control Plane features.
```

Comment:
```suggestion
You can use the `save_artifact` utility function to save your data assets as ZenML artifacts. Moreover, if your step has the Model context configured in the `@pipeline` or `@step` decorator it will be automatically linked to it, so you can get easy access to it using the Model Control Plane features.
```